CHANGES.txt

```diff
@@ -13,6 +13,14 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
         - Whatever John Doe did.
 
+  From Mats Wichmann:
+    - Tweak definition of default site_scons paths on Windows to conform
+      to conventions, drop a Py2 workaround, and pick as better "system" path
+      (old one remains supported).
+    - Fix testsuite to work on Windows systems where there is no usable
```

Comment:
Add a note that Visual Studio Code is the usual suspect when this association get's broken?

Reply:
Added.